R/tmb-methods.R

```diff
@@ -166,33 +166,112 @@ print.mmrm_tmb <- function(x,
                            ...) {
   cat("mmrm fit\n\n")
 
-  n_subjects <- x$tmb_data$n_subjects
-  n_timepoints <- x$tmb_data$n_visits
-  n_obs <- length(x$tmb_data$y_vector)
-  cov_type <- x$formula_parts$corr_type
-  n_theta <- length(x$theta_est)
+  components <- component(x, c(
```

Comment:
here instead of saving everything in large list I think it is better readable to just call each component where needed below.

Reply:
done